x/ibc/03-connection/keeper/keeper_test.go

```diff
@@ -67,18 +67,20 @@ func (suite *KeeperTestSuite) SetupTest() {
 }
 
 // nolint: unused
-func queryProof(chain *TestChain, key []byte) (commitmenttypes.MerkleProof, uint64) {
+func queryProof(chain *TestChain, key []byte) ([]byte, uint64) {
 	res := chain.App.Query(abci.RequestQuery{
 		Path:   fmt.Sprintf("store/%s/key", storeKey),
 		Height: chain.App.LastBlockHeight(),
 		Data:   key,
 		Prove:  true,
 	})
 
-	proof := commitmenttypes.MerkleProof{
+	merkleProof := commitmenttypes.MerkleProof{
 		Proof: res.Proof,
 	}
 
+	proof, _ := chain.App.AppCodec().MarshalBinaryBare(&merkleProof)
```

Comment:
Is it intentional to ignore the error here?

Reply:
yea, I'm planning on deleting this dup func within the next week in favor of importing one centralized version of querying. 

Checking the error would require adding in a `*testing.T` to `TestChain` which was more engineering debt than I'd like for the benefit

Reply:
Alright, that makes sense.

> I'm planning on deleting this dup func within the next week in favor of importing one centralized version of querying.

Have you opened an issue for that already or are you planning to do so as soon as this is merged in?

Reply:
yup, its apart of the #5558 meta-issue